tests/migrations/test_update_mlwh_with_legacy_samples.py

```diff
@@ -0,0 +1,162 @@
+import sys
+import traceback
+from migrations.helpers.mlwh_samples_update_helper import (
+    valid_datetime_string,
+    update_mlwh_with_legacy_samples,
+)
+from crawler.constants import (
+    FIELD_ROOT_SAMPLE_ID,
+    FIELD_RNA_ID,
+    FIELD_RESULT,
+    FIELD_DATE_TESTED,
+    FIELD_LAB_ID,
+    FIELD_SOURCE,
+    FIELD_PLATE_BARCODE,
+    FIELD_CREATED_AT,
+    FIELD_UPDATED_AT,
+    MLWH_TABLE_NAME,
+    MONGO_DATETIME_FORMAT,
+)
+import pytest
+from datetime import (datetime, timedelta)
+
+def print_exception(e: Exception) -> None:
+    print(f'An exception occurred, at {datetime.now()}')
+    e = sys.exc_info()
+    print(e[0]) # exception type
+    print(e[1]) # exception message
+    if e[2]: # traceback
+      traceback.print_tb(e[2], limit=10)
```

Comment:
So I think we could just import logger and do `logger.exception(e)` - did you try it?

Reply:
using pytest.fail() instead